automated-testing/unit-testing/custom-connector.md

```diff
@@ -0,0 +1,11 @@
+# Custom Connector Testing
+
+When developing Custom Connectors to put data into the Power Platform there are some strategies you can follow:
+
+## Unit Testing
```

Comment:
I am not sure these are really considered unit tests seeing as it seems to be testing the structure of the API itself and not the behavior or state of the API.  Perhaps its actually closer to UI tests, or a part of an e2e suite? Would be curious what others thing!

One idea would be that we define this as a subset of a broader test type where we validate the schema or our APIs and validating a custom connector is one possible avenue of that

Reply:
Huuuum, I am not sure if this is e2e, since we have another steps to perform the e2e tests in this scenario, actually. And its not UI testing as well, since there is no UI for this.

I like your suggestion of defining it as a subset of a broader test type. How do we want to approach that?

Reply:
I am not sure where we should put it :) @alkofahi, do you have an idea of where this sort of documentation belongs?

Reply:
Guilherme, thanks for your contribution. To me, this is coming as an example/a case study of how to address a specific challenge rather than a test type per se. I think we need to add a new section for case studies and align on how to structure those documents. Especially that we also have Oliver's PR that is discussing their learnings from using DevTest patterns.

Reply:
@alkofahi very well observed. The specific challenge in this case is directly related to the low-code characteristics of custom connectors, what limit the scope of validation. As it is probably applied to any custom connector, maybe a good section to consider is any with tech related test approaches. Please let me know your thoughts.

Reply:
@madossan01 - it seems Guilherme moved out of CSE, do you have enough context to address the comments on this PR? I think it needs additional context on the challenges faced and how the suggestions would help resolve them.